packer/scripts/Artifacts/MsSql/Structure/Ods/2110-CreateProcedureLoadSpedChildCount.sql

```diff
@@ -0,0 +1,159 @@
+        , s.FirstName
+        , MiddleName = COALESCE(s.MiddleName,'')
+        , s.Birthdate
+        , Age = (YEAR(@October1) - YEAR(s.Birthdate)) - (
+            CASE WHEN MONTH(s.birthdate) < MONTH(@October1) THEN 0
+            WHEN MONTH(s.birthdate) = MONTH(@October1) AND DAY(s.Birthdate) = DAY(@october1) THEN 0 ELSE 1 END
+            )
+        , DisabilityCode = d_disability.CodeValue
+        , DisabilityDescription = d_disability.ShortDescription
+        , SpeechLanguageTherapyServices = COALESCE(spas.SL,'N')
+        , OccupationalTherapyServices = COALESCE(spas.OT,'N')
+        , SettingCode = ISNULL(d_Setting.CodeValue,'')
+        , SettingDescription = ISNULL(d_Setting.ShortDescription,'')
+        , PartBC = ISNULL(d_Program.ShortDescription,'')
+        , SpecialEducationPercentage = CAST(CASE WHEN ISNULL(schoolHoursPerWeek,0.0) > 0.0 THEN CONVERT(INT,ROUND(100.0*ISNULL(SpecialEducationHoursPerWeek,0.0)/ISNULL(schoolHoursPerWeek,0.0),0)) ELSE 0.0 END AS INT)
+        , AlternateAssessment = CASE ssepae.ToTakeAlternateAssessment WHEN 1 THEN 'Y' ELSE 'N' END
+        , race.reporting_race as RaceCode
+        , race.reporting_race_desc as RaceDescription
+        , GradeLevel = ISNULL(d_gradelevel.CodeValue,'')
+        , GenderCode = COALESCE(LEFT(st.ShortDescription,1),'M')
+        , GenderDescription = COALESCE(st.ShortDescription,'Male')
+        , EnglishLearnerCode = CASE WHEN el.RedesignatedEnglishFluent = 1 THEN '13'
+                                WHEN el.englishLearnerParticipation = 1 THEN '11'
+                                WHEN el.englishLearnerParticipation = 0 THEN '12'
+                                ELSE '14' END
+        , EnglishLearnerDescription = CASE WHEN el.RedesignatedEnglishFluent = 1 THEN 'Redesignated English Fluent'
+                                        WHEN el.englishLearnerParticipation = 1 THEN 'English Learner Eligible And Participating'
+                                        WHEN el.englishLearnerParticipation = 0 THEN 'English Learner Eligible Not Participating'
+                                        ELSE 'Not an English Learner' END
+        , ISNULL(d_residentstatus.CodeValue,'') as ResidenceStatusCode
+        , ISNULL(d_residentstatus.ShortDescription,'') as ResidenceStatusDescription
+        , UpdateDate = GETDATE()
+    FROM edfi.GeneralStudentProgramAssociation spa WITH(NOLOCK)
+    INNER JOIN reporting.GetSpedBeginDate(@October1) msbd ON
+        spa.StudentUSI = msbd.StudentUSI
+        AND spa.BeginDate = msbd.BeginDate
+    INNER JOIN edfi.Student s WITH(NOLOCK) ON
+        spa.StudentUSI = s.StudentUSI
+    INNER JOIN edfi.StudentSchoolAssociation ssa WITH(NOLOCK) ON
+        spa.StudentUSI = ssa.StudentUSI
+    INNER JOIN sk.StudentSchoolAssociationExtension ssae WITH(NOLOCK) ON
+        ssa.EntryDate = ssae.EntryDate
+        AND ssa.SchoolId = ssae.SchoolId
+        AND ssa.StudentUSI = ssae.StudentUSI
+    INNER JOIN edfi.Descriptor d_residentstatus ON
```

Comment:
@jamessmckay change this to LEFT JOIN as in the membership report. Sample data doesn't have ResidencyStatusDescriptorId populated.

Reply:
done